pkg/controller/operators/catalog/operator.go

```diff
@@ -2252,6 +2281,10 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 					}
 					rb.SetOwnerReferences(updated)
 					rb.SetNamespace(namespace)
+					if rb.Labels == nil {
+						rb.Labels = map[string]string{}
+					}
+					rb.Labels[install.OLMManagedLabelKey] = install.OLMManagedLabelValue
```

Comment:
I am seeing this idiom so much, it almost makes sense to create a utility API to initialize the labels and then add the label.

e.g. 
```
addlabel(&rbLabels, install.OLMManagedLabelKey, install.OLMManagedLabelValue)